src/main/java/gov/cabinetofice/gapuserservice/security/WebSecurityConfig.java

```diff
@@ -47,6 +47,7 @@ public WebSecurityCustomizer webSecurityCustomizer() {
                 "/login",
                 "/v2/redirect-after-login",
                 "/v2/login",
+                "/v2/notice-page",
```

Comment:
This isn't actually required here as we will want to protect this behind auth.

For local development you can set the following properties:

debug.ignoreJwt=true
spring.profiles.active=LOCAL